src/base/bittorrent/session.cpp

```diff
@@ -4492,6 +4504,20 @@ void Session::handleFileErrorAlert(const lt::file_error_alert *p)
     m_recentErroredTorrentsTimer->start();
 }
 
+void Session::handleReadPieceAlert(const lt::read_piece_alert *p)
+{
+    lt::error_code ec;
+    if (ec) {
```

Comment:
Remove the `ec` variable declaration and use:
`if (p->error)`

Reply:
Looks like p->error doesn't work for RC 1_1.